pkg/operator/starter.go

```diff
@@ -438,6 +438,18 @@ func prepareOauthOperator(controllerContext *controllercmd.ControllerContext, op
 	return nil
 }
 
+type encryptionProvider []schema.GroupResource
```

Comment:
/hold

I'm going to move it to library-go.

A static provider will be used by oas, cao and kas-o

Reply:
https://github.com/openshift/library-go/pull/992

Reply:
merged.

/hold cancel